bitcoin/src/address.rs

```diff
@@ -158,6 +162,11 @@ impl From<bech32::Error> for Error {
     fn from(e: bech32::Error) -> Error { Error::Bech32(e) }
 }
 
+#[doc(hidden)]
+impl From<PushBytesError> for Error {
```

Comment:
There's a much better solution I proposed in constification PR (now it's no longer up, will re-add). Anyway valid addresses will never break this so we can even `unwrap()`

Reply:
Good observation, in `FromStr` we need it still but not if my other `WitnessProgram` PR gets in :) For now, since this PR is not mergable i just added `expect("TODO: ...")` and removed the error variant.